flock_agent/gui/bootstrap.py

```diff
@@ -18,26 +19,24 @@ class Bootstrap(object):
 
     def __init__(self, common):
         self.c = common
-        self.c.log("Bootstrap", "__init__")
+        logger = logging.getLogger("Bootstrap.__init__")
+        logger.debug("")
 
     def go(self):
         """
         Go through all the bootstrap steps
         """
         platform = Platform.current()
 
-        self.c.log("Bootstrap", "go", "Bootstrapping Flock Agent", always=True)
+        logger = logging.getLogger("Bootstrap.go")
+        logger.warning("Bootstrapping Flock Agent")
```

Comment:
I think we should user `logger.info` for this, since it happens every time, and just reserve `logger.warning` for subprocess calls